week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -1,24 +1,154 @@
+/*
+ Before we go the big story; we will introduce more string methods.
+ Some of the methods you're using in Array have similar ones with strings.
+
+ Methods like : IndexOf, Include, Search, Slice , Spilt and more.
+ 
+ You can always google how a method of a string works!
+ Here are links to some of those:
+  - https://www.w3schools.com/js/js_string_methods.asp
+  - https://javascript.info/s
+  tring#quotes
+
+
+ Now let's do this small exercise
+ 
+ Using string methods update the checkCodeIsThere() function
+  - The function will have a string as a paramter
+  - The function should check if the string has the word "code" exists in the string
+  - If it does exist, return the index of it, if not return "Not found"
+
+ Hint: search for string methods like Includes and IndexOf.
+*/
+
+function checkCodeIsThere(stringText) {
+  let magicWord = "code";
```

Comment:
You're not using your `magicWord` variable in your function at the moment. You declare it and then you don't use it anywhere else.